pages/common/jest.md

```diff
@@ -7,15 +7,19 @@
 
 `jest`
 
-- Run only tests with a name that matches the regex pattern:
+- Run tests from the test files that match the regex patterns:
+
+`jest {{test_file1}} {{test_file2}}`
```

Comment:
Since we are mentioning the word regex, it might be useful to use a regex like `file*.js`.

Reply:
Perhaps `jest {{*.test.js}}` as the example then?

Reply:
If it's a regular expression,  don't you mean `file.*.js`, @agnivade?

Reply:
Yes, I didn't focus on the details :P Just wanted to get my point across. @garyking - what you say is fine.

Reply:
`jest {{*.test.js}}` is still a glob. It would need to be `jest {{.*.test.js}}` to be a regular expression.

Reply:
I'll just leave it as `test_file1` and `test_file2`, just like the `spec_name`.